doc/cookbook/source/examples/classifier/multiclass_logisticregression.rst

```diff
@@ -0,0 +1,43 @@
+===============================
+Multi-class Logistic Regression
+===============================
+
+Logistic regression is a binary classifier. The samples are predicted to be one case or not based on the probability calculated by feature matrix and logistic function:
+
+.. math::
+ 
+    z = \alpha +  \mathbf{\beta x} = \alpha + \beta_{1}x_{1} + ... + \beta_{d}x_{d}
```

Comment:
You have to say what these letters mean, otherwise the math is not helpful.
Also, if you could state the loss function that is minimised, that would be great